frontend/src/_layouts/base.html

```diff
@@ -57,8 +61,8 @@
         <nav
           x-cloak
           @keydown.escape="open = false"
-          :data-open="open"
-          class="fixed top-0 right-0 z-40 flex h-screen w-64 translate-x-full flex-col items-start gap-4 overflow-hidden border-l border-l-gray-300 bg-white p-4 pt-14 text-sm font-semibold uppercase text-stone-900 transition-transform data-[open]:translate-x-0 md:relative md:top-auto md:h-auto md:w-auto md:max-w-none md:translate-x-0 md:flex-row md:border-l-0 md:bg-none md:p-0 lg:gap-x-10"
+          :class="open ? 'translate-x-0': 'translate-x-full'"
+          class="fixed top-0 right-0 z-40 flex h-screen w-64 flex-col items-start gap-4 overflow-hidden border-l border-l-gray-300 bg-white p-4 pt-14 text-sm font-semibold uppercase text-stone-900 transition-transform md:relative md:top-auto md:h-auto md:w-auto md:max-w-none md:translate-x-0 md:flex-row md:border-l-0 md:bg-none md:p-0 lg:gap-x-10"
```

Comment:
I think we don't need `md:translate-x-0` anymore because there is no `translate-x-full` to override.

Reply:
`md:translate-x-0` kept as `translate-x-full` is overriding from alpine `open=false`